packages/commerce-sdk-react/README.md

```diff
@@ -1,24 +1,51 @@
 # Commerce SDK React
 
-A library of React hooks for fetching data from Salesforce B2C Commerce.
+<p align="center">
+A collection of React hooks for <b>fetching</b>, <b>caching</b>, and <b>mutating data</b> from the <b><a href="https://developer.salesforce.com/docs/commerce/commerce-api/overview">Salesforce B2C Commerce API</a></b> (SCAPI).
+The library contains declarative, always-up-to-date auto-managed <b>queries</b> and <b>mutations</b> for a more convenient developer experience.
+</p>
 
-## Documentation
+## 🎯 Features
 
-The full documentation for PWA Kit and Managed Runtime is hosted on the [Salesforce Developers](https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/overview) portal.
+- [react-query](https://tanstack.com/query/latest/docs/react/overview) and [commerce-sdk-isomorphic](https://github.com/SalesforceCommerceCloud/commerce-sdk-isomorphic) integration
+- Shopper authentication & token management via [SLAS](https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-login)
+- SSR data fetching React Hooks
+- Phased Launch support ([plugin_slas](https://github.com/SalesforceCommerceCloud/plugin_slas) compatible)
+- [TypeScript](https://www.typescriptlang.org/) Support
+- Multi-site and locale/currency support
+- Request deduplication
+- Built-in caching for easy state management
+  - automatic cache invalidations/updates via the library's built-in mutations
+  - automatic cache key generation
 
+## ⚙️ Installation
```

Comment:
Is adding a **Requirements** section with the Node engines requirements useful?

Are we missing a BSD 3-Clause license Information section?

Reply:
Hmmmmm, I don't believe node requirements is super important for this library. Unless someone ran into issues with a specific node version.

The license is [here](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/develop/LICENSE), do we need it in the README as well? @adamraya